readme.md

```diff
@@ -180,7 +185,11 @@ customer or a charge, but only used once.
 	var tokenService = new StripeTokenService();
 	StripeToken stripeToken = tokenService.Create(myToken);
 
-Tokens are very useful if you don't want to store the customers credit card information on your servers (pci compliance).
+You should never create tokens with Stripe.net with real credit cards, since
+giving your server access to unredacted card data greatly increases your
+PCI-DSS compliance burden.
```

Comment:
this line is fine to mention, but the blanket statement of never isn't necessarily better or easier. See previous comment about temporarily storing credit cards.